epysurv/simulation/point_source.py

```diff
@@ -94,5 +94,7 @@ def simulate(
         return (
             simulated_as_frame.pipe(add_date_time_index_to_frame)
             .rename(columns={"observed": "n_cases", "state": "is_outbreak"})
-            .assign(n_outbreak_cases=lambda df: df["n_cases"] * df["is_outbreak"])
+            .assign(
+                n_outbreak_cases=lambda df: df["n_cases"] * df["is_outbreak"]
```

Comment:
Is it reasonable (based on the model) to assume that all cases are outbreak cases if there is an outbreak. If not we should maybe just leave the `n_outbreak_cases` column out here as well.

Reply:
No it is not reasonable enough. I will omit n_outbreak_cases.

Reply:
I think this is the only remaining open point, that keeps us from merging this.